app/src/components/keyboardNavigation.js

```diff
@@ -0,0 +1,64 @@
+import { goToNextSlide, goToPrevSlide } from "../action_creators";
+import { Component } from "./_componentBase";
+
+export class KeyboardNavigation extends Component {
+  constructor(props) {
+    super(props);
+  }
+
+  init() {
+    super.checkForStore();
+    this.slides = this.element.querySelectorAll(".slide");
+
+    this.bindEvents = this.bindEvents.bind(this);
+    this.handleKeyDown = this.handleKeyDown.bind(this);
+    this.maybeGoToNextSlide = this.maybeGoToNextSlide.bind(this);
+    this.maybeGoToPrevSlide = this.maybeGoToPrevSlide.bind(this);
+
+    this.bindEvents();
+  }
+
+  bindEvents() {
+    document.addEventListener("keydown", this.handleKeyDown);
+  }
+
+  removeEvents() {
+    document.removeEventListener("keydown", this.handleKeyDown);
+  }
+
+  cleanUp() {
+    this.removeEvents();
+  }
+
+  handleKeyDown(event) {
+    switch (event.code) {
```

Comment:
add a guard clause: `if (!event.target.matches("body")) return;` so that this event isn't intercepted by the search address input when selecting an item from the `<datalist>`